tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -597,6 +598,15 @@ def _send_reset_request(headers: dict) -> None:
     )
     response.raise_for_status()
 
+    headers_to_ignore = "Authorization, x-ms-client-request-id, x-ms-request-id"
+    add_remove_header_sanitizer(headers=headers_to_ignore)
+    set_custom_default_matcher(excluded_headers=headers_to_ignore)
+
+    client_id = os.environ.get("AZURE_CLIENT_ID", "00000000-0000-0000-0000-000000000000")
+    client_secret = os.environ.get("AZURE_CLIENT_SECRET", "00000000-0000-0000-0000-000000000000")
+    add_general_regex_sanitizer(regex=client_id, value="00000000-0000-0000-0000-000000000000")
+    add_general_regex_sanitizer(regex=client_secret, value="00000000-0000-0000-0000-000000000000")
```

Comment:
sub_id and tenant_id are also mostly used so I think it is better to add them here too:
![image](https://user-images.githubusercontent.com/70930885/182745568-0417d106-e570-4bb8-a136-8077e3882b95.png)

Reply:
I think this base set of 4 is fine. Thanks for the heads up @msyyc